app/layout/layout.phtml

```diff
@@ -63,7 +63,7 @@
 		<meta name="robots" content="noindex,nofollow" />
 <?php } ?>
 	</head>
-	<body class="<?= Minz_Request::actionName() ?>">
+	<body class="<?= Minz_Request::actionName() ?> theme-<?= strtolower(str_replace([' ', '_', '.', '<', '>', '&', '"', '\''], '-', html_entity_decode(FreshRSS_Context::userConf()->theme))) ?>">
```

Comment:
We do not use HTML entities, except for the few XML ones.
So I do not think it is necessary, but if it is, the lighter and safer `htmlspecialchars_decode()` should be sufficient